src/nvim/ui.c

```diff
@@ -128,6 +129,19 @@ void ui_builtin_start(void)
 #endif
 }
 
+uint64_t ui_client_start(char *server_name, int argc, char **argv, bool pass_stdin)
+{
+  ui_comp_detach(uis[1]);  // Bypassing compositor in client
+  uint64_t rv = tui_ui_client_init(server_name, argc, argv, pass_stdin);
+  return rv;
+}
+
+UI* get_ui_by_index(int idx)
```

Comment:
```suggestion
UI* ui_get_by_index(int idx)
```

Almost always better to have common module prefix.